src/shared/models/ipc/ipc-routes.ts

```diff
@@ -43,6 +41,12 @@ export interface IpcChannelMapping {
     "bsv-search-playlist": {request: PlaylistSearchParams, response: BsvPlaylist[]};
     "bsv-get-playlist-details-by-id": {request: {id: string, page: number}, response: BsvPlaylistPage};
 
+    /* ** bs-installer-ipcs ** */
+    "bs-installer.folder-exists": { require: void, response: boolean };
```

Comment:
You wrote `require` here
```suggestion
    "bs-installer.folder-exists": { request: void, response: boolean };
```